Projects/BricKuber/brickuber_lib.py

```diff
@@ -232,13 +267,19 @@ def Moves(self, cmds):
     
     # Use the camera to read the RGB colors for each of the 9 squares on the face
     def CameraReadFaceColors(self, face):
-        commands.getstatusoutput(('raspistill -w 300 -h 300 -t 1 -o /tmp/BricKuber_%s_face.jpg' % face))
+        debug_print_commands("START: Read Face Colors: " + str(face))
+        # commands.getstatusoutput(('raspistill -w 300 -h 300 -t 1 --sharpness -100 --awb auto --imxfx cartoon -o /tmp/BricKuber_%s_face.jpg' % face))
+        commands.getstatusoutput(('raspistill -w 300 -h 300 -t 1 --imxfx cartoon -o /tmp/BricKuber_%s_face.jpg' % face))
+        debug_print_commands("Picture taken")
         raw_result = commands.getstatusoutput(('rubiks-cube-tracker.py --filename /tmp/BricKuber_%s_face.jpg' % face))[1]
         raw_result = raw_result.split("\n{")[1]
         raw_result = raw_result.split("}")[0]
         raw_results = raw_result.split("[")[1:]
+ 
+        
         for c in range(9):
             raw_results[c] = raw_results[c].split("]")[0]
+        print(raw_results)
```

Comment:
Why print this? If being printed, it should probably be in a debug print, right?

Reply:
I think we leave it in, it looks cool in my opinion.